### DISCUSSION TITLE
Bar Chart: is it possible to set a default selection value?

### DISCUSSION THREAD

**Discussion by yepMad** [apexcharts/apexcharts.js#3162] [Q&A] [answered]
I would like to know if it is possible to set a default selection (or active) value. I can't find any function or property.

The selection I'm referring to is the one that happens when you "click" on a bar.
![image](https://user-images.githubusercontent.com/22564368/169728261-3accf682-12a6-4877-911c-2549113f26f0.png)

**Comment by MarioPerini** [comment, marked as answer]
Did you try https://apexcharts.com/docs/methods/#toggleDataPointSelection ?

**Reply by yepMad** [reply]
Sorry! I miss that. Thanks.

**Reply by MarioPerini** [reply]
No reason for "Sorry" glad to help 😉
